### PR TITLE
DDF-5373 Remove manual metacard creation view

### PR DESCRIPTION
**Pull request by zta6**
#### What does this PR do?
Removes the ability for users to manually create metacards to be added to their lists from within the workspaces view. 
This feature was being used by downstream projects but is no longer desired. 

#### Who is reviewing it? 

@gordocanchola 
@andrewzimmer 
@willwill96 
@hayleynorton 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining
@djblue

#### How should this be tested?

Build
Profile:install
Create a list and click the button circled in the first screenshot below. 
Verify that the modal that opens has only one tab, `import`, and no tab called `manual`, as shown in the second screenshot.

#### What are the relevant tickets?
Fixes: #5373 

#### Screenshots

button
modal

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/component/tabs/list-add/tabs-list-add.js [hunk 1]
```diff
@@ -15,13 +15,11 @@
 
 const Tabs = require('../tabs')
 const IngestView = require('../../ingest/ingest.view.js')
-const BuilderView = require('../../builder/builder.view.js')
 
 module.exports = Tabs.extend({
   defaults: {
     tabs: {
       Import: IngestView,
-      Manual: BuilderView,
```
Comment (millerw8) [inline ui/packages/catalog-ui-search/src/main/webapp/component/tabs/list-add/tabs-list-add.js]: Should we still be using tabs here?

**Comment by zta6** [comment]
@andrewkfiedler I was told @vinamartin and @stevenmalmgren are replacing the view and I shouldn't delete it yet, but if that has changed I can certainly delete it. Also should I be refactoring this to something other than tabs?

**Comment by zta6** [comment]
build now

**Comment by zta6** [comment]
build now

**Comment by andrewzimmer** [comment]
Hero Successful 🎉 
- verified no manual tab 
Screen Shot 2019-10-03 at 1 42 58 PM

**Pull request merged**